raft.go

```diff
@@ -1739,7 +1774,7 @@ func (r *Raft) electSelf() <-chan *voteResult {
 		if server.Suffrage == Voter {
 			if server.ID == r.localID {
 				// Persist a vote for ourselves
-				if err := r.persistVote(req.Term, req.Candidate); err != nil {
+				if err := r.persistVote(req.Term, req.Addr); err != nil {
```

Comment:
`req.Addr` is the new field, and all the other code checks the version. Do we need to have a version check here as well in case the code is still running in V3 mode?

Reply:
It's not coming form the wire, we just populated it line 1746. So no need for a check here.